sklearn/preprocessing/imputation.py

```diff
@@ -177,6 +178,10 @@ def fit(self, X, y=None):
                                                    self.strategy,
                                                    self.missing_values,
                                                    self.axis)
+            invalid_mask = np.isnan(self.statistics_)
```

Comment:
Is this convention that `nan` marks all-missing columns something established by documentation and/or tests?